system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -90,7 +90,13 @@ typedef long suseconds_t;
 
 
 #if defined(__NEED_pthread_attr_t) && !defined(__DEFINED_pthread_attr_t)
+#ifdef __EMSCRIPTEN__
```

Comment:
Yes, though the file has been copied from musl upstream, so all the other changes are marked under #ifdef, so following suit to keep info where we diverge.